python/ray/_private/worker.py

```diff
@@ -2273,7 +2273,11 @@ def get(
         object_refs: Object ref of the object to get or a list of object refs
             to get.
         timeout (Optional[float]): The maximum amount of time in seconds to
-            wait before returning.
+            wait before returning. Set this to None will block until the
+            corresponding object becomes available.
+
+            WARNING: In future ray releases, set ``timeout=0`` will return the object
```

Comment:
It would be good to add a little clarification here, similar to the blow (feel free to change the wording):

```
WARNING: In future ray releases ``timeout=0`` will return the object
immediately if it's available, else raise GetTimeoutError in accordance with
the above docstring. The current behavior of blocking until objects become
available of ``timeout=0`` is considered to be a bug, see
https://github.com/ray-project/ray/issues/28465.
```